pow/src/types.rs

```diff
@@ -56,6 +56,7 @@ pub struct MinerConfig {
 	/// Base address to the HTTP wallet receiver
 	pub wallet_listener_url: String,
 
+	/// TODO: should be removed or renamed, due to the lack of burning?
```

Comment:
Burning fees and burning rewards are different (confusing) concepts.
Burning a reward is used in testing - instead of "sending" the reward to a wallet we use a random key and simply make it unspendeable aka burnt.